app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -61,34 +58,27 @@ open class BpkSnapshotTest(private val tags: List<Any> = emptyList()) : Screensh
       view = view,
       background = background,
       padding = padding,
-      wrapView = wrapView,
     )
     wrappedView.measure(measureSpec(width), measureSpec(height))
     compareScreenshot(
       view = wrappedView,
-      widthInPx = wrappedView.measuredWidth,
-      heightInPx = wrappedView.measuredHeight,
+      widthInPx = width?.let { dpToPx(it) } ?: wrappedView.measuredWidth,
```

Comment:
always use the exact size if specified, otherwise measured one. this shouldn't change any tests, but is more consistent with what we're doing